dev/Installer/src/help.cpp

```diff
@@ -0,0 +1,18 @@
+#include "pch.h"
+#include "help.h"
+
+void show_help()
+{
+    std::wcout << L"Installs the Microsoft Project Reunion framework." << std::endl;
+
+    // Get the program name for the module, since argv[0] is unreliable if launched from CreateProcess.
+    wchar_t programName[MAX_PATH];
+    if (::GetModuleFileNameW(GetModuleHandle(NULL), programName, MAX_PATH))
+    {
+        printf("%ls\n", programName);
+    }
+
+    printf("Usage:\n");
```

Comment:
[External]
Probably want to excise all printf usage and ansi strings in the project, at some point.

Reply:
Move everything to `std::wcout`

Reply:
https://kennykerr.ca/2014/12/08/in-defense-of-printf/ 😉

Reply:
```
int x = 123456;
std::cout << std::hex << x;
```
vs
```
int x = 123456;
printf("%x", x);
```
Printf easily wins on clarity and maintainability even in simple cases let alone real world uses. The perf benefits are just the cherry on top :-)

Reply:
I moved it all over to std::wcout and std::wcerr, though I am sympathetic to the plight of printf and its simplicity and readability.